config/ranks/military.txt

```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
+Head of Security=LT
+Chief Engineer=LTCDR
+Research Director=LT
+Chief Medical Officer=LTCDR
```

Comment:
Okay I'll buy that doctors are usually officers but you still need to fix the abbreviation.